sql/migrations/20220501213609_world.sql

```diff
@@ -5012,9 +5012,27 @@ UPDATE `pool_gameobject` SET `pool_entry` = 1274 WHERE `guid` IN (397176, 397177
 DELETE FROM `pool_template` WHERE  `entry`=1319;
 DELETE FROM `pool_template` WHERE  `entry`=1245;
 
--- LOOK AT ALL THE CUSTOM ORE OBJECTS
--- SELECT * FROM gameobject WHERE id IN (SELECT entry FROM gameobject_template WHERE NAME LIKE "%Vein%") AND rotation2 = 0
--- SELECT * FROM gameobject WHERE id IN (SELECT entry FROM gameobject_template WHERE NAME LIKE "%Deposit%") AND rotation2 = 0
+/*
+LOOK AT ALL THE CUSTOM ORE OBJECTS
+SELECT * FROM gameobject WHERE id IN (SELECT entry FROM gameobject_template WHERE NAME LIKE "%Vein%") AND rotation2 = 0
+SELECT * FROM gameobject WHERE id IN (SELECT entry FROM gameobject_template WHERE NAME LIKE "%Deposit%") AND rotation2 = 0
+*/
+
+/*
+NOTE: CMANGOS HAS LOTS MORE ORES HERE (COMMITED DIRECTLY TO CLASSIC SO NOT BACK-PORTED)

```

Comment:
+1

Reply:
Do you know what's the source of the data?

Reply:
Looking at our recent classic zone reworks, i can say that alot of spawns are faulty and had to be readjusted.
and thats what i expect of anyone committing to mangos, have brains and just dont go half way of what you're looking at peronally.
I hope people really try to do their best to make the content the best.